schulcloud/Notifications.swift

```diff
@@ -17,26 +17,27 @@ import FirebaseInstanceID
 import FirebaseMessaging
 
 class SCNotifications {
-    
+
     static func checkRegistration() -> Future<Void, SCError> {
```

Comment:
This doesn't do any work anymore, can this be removed?

Reply:
I left the old code as a comment to make the fix easier.